### DISCUSSION TITLE
Call me up to support in some capacity.

### DISCUSSION THREAD

**Discussion by glebite** [WeAreMahsaAmini/Core#361] [Introductions]
I have a few tools and such that could be used/distributed - mainly education stuff and one quasi-surveillance tool in my repos:

- [Dictation program for learning English](https://github.com/glebite/dikteh)
- [Farsi typing tutor - leave the world of stickers](https://github.com/glebite/keyboarder)
- [A faster insecam scraper](https://github.com/glebite/ihaveaproblem)

I have a couple of other thoughts of dusting off an old ping/network monitor tool to detect network degradation/access to important sites within Iran.  

Just let me know - check out my profile and I'd love to help in any capacity!

**Comment by glebite** [comment]
I just added up some code to monitor the accessibility of university sites in Iran - could be expanded to send out alerts.  

[https://github.com/glebite/expansion](https://github.com/glebite/expansion)

**Comment by Pink3lephant** [comment]
Hi @glebite 

Nice to hear from you. We are working on a project to distribute knowledge on how to create VPN servers that actually work (think of it as meta-Outline, where you can setup several protocols).
Can you please have a look and let me know if you're interested in helping out? 

🐜  Current website: [Antyfilter.com](https://Antyfilter.com)
👨‍💻 [Github](https://github.com/Antyfilter)


You can email me at `freeinternettech` [at] gmail.
